torchdp/compute_dp-sdg_privacy.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
typo in file name

Reply:
Maybe you suggest `torchdp/compute_dp_sdg_privacy.py ` (with `_` and no `-` sign) instead of `torchdp/compute_dp-sdg_privacy.py ` ? 
It doesn't matter for me, so feel free to adapt it (I meant a compound name, but maybe I'm not aware about naming conventions).

Reply:
Note: if this file name is updated, it has to be modified too in README.md content.

Reply:
Hi @jmg-74, no I meant `compute_dp-sgd_privacy.py` instead of `compute_dp-sdg_privacy.py`
s**g**d  instead of s**d**g

Reply:
Oops ! Maybe am I dyslexic ? Of course you're right, sorry for wasting your time.

Reply:
No worries! No need to be too harsh on yourself. :)
Please let us know once you are ready with this change and we can merge this.
Thanks for your contribution :)

Reply:
Thank you :)  `compute_dp_sgd_privacy.py` sounds perfect to me, since `_` it allows `import` with no `-`.
But I 'm not sure I've done it correctly. If not, maybe could you do this for me please ? Or tell me how to ? Thanks !

Reply:
Thank you :)  `compute_dp_sgd_privacy.py` sounds perfect to me, since `_` it allows `import` with no `-`.
But I 'm not sure I've done it correctly. If not, maybe could you do this for me please ? Or tell me how to ? Thanks !